lib/remote_record/base.rb

```diff
@@ -29,6 +29,10 @@ def get
       raise NotImplementedError.new, '#get should return a hash of data that represents the remote record.'
     end
 
+    def self.all
+      raise NotImplementedError.new, '#all should return an array of hashes of data that represent remote records.'
+    end
```

Comment:
I understand having an `all` method on the reference (fetch all the objects in my DB) but I don't understand having an `all` method here. What objects would be included in all?

Reply:
This should point to an endpoint that returns multiple remote records, e.g. `api.github.com/V3/users`. The array of data that's returned then gets used to initialize references with the fetched attributes, saving you from making a request per record.